aip/general/0194.md

```diff
@@ -28,7 +28,8 @@ Clients **should not** automatically retry requests in which repeated runs
 would cause unintended state changes.
 
 **Note:** This AIP does not cover client streaming or bi-directional streaming.
-**Note:** Client side retry behavior for the client libraries is found in [AIP-4221](../client-libraries/4221.md).
+
+**Note:** Client side retry behavior for the client libraries is found in [AIP-4221](../client-libraries/4221).
```

Comment:
```suggestion
**Note:** For client side retry behavior in the client libraries: see AIP-4221

Reply:
Nit: don't forget the period at the end

LGTM your other text was fine too but I guess this is a little more succinct :)

Reply:
Fixed! Love a good nit. And lol thank you - I was just trying to make it match the other page you shared to maximize the chance of its success 😁